plugins/cliconf/vyos.py

```diff
@@ -194,12 +194,18 @@ def get(
             check_all=check_all,
         )
 
-    def commit(self, comment=None):
-        if comment:
-            command = 'commit comment "{0}"'.format(comment)
+    def commit(self, comment=None, confirm=None):
+        if confirm:
+            if comment:
+                command = 'commit-confirm {0} comment {1}'.format(confirm, comment)
+            else:
+                command = 'commit-conform {0}'.format(confirm)
```

Comment:
Should it be via **i** `commit-confirm`?

Reply:
Yes, absolutely!